modular-docs-manual/files/TEMPLATE_PROCEDURE_doing-one-procedure.adoc

```diff
@@ -53,13 +49,15 @@ Procedure modules should include the steps that users perform and address user m
 ////
 Delete this section if it does not apply to your module. Provide the user with verification methods for the procedure, such as expected output or commands that confirm success or failure.
 ////
+. Provide example of expected command output or a pop-up window that the user receives when the procedure is successful.
```

Comment:
```suggestion
. Provide an example of expected command output or a pop-up window that the user receives when the procedure is successful.
```

Reply:
... of the expected command output ...

However, I would go with something a bit different:
`Provide a command output or a screenshot of a result in GUI that the user receives when the procedure is successful. Consider trimming the output and cropping the screenshot to contain only the relevant parts.`